### PR TITLE
Usecases

### PR DESCRIPTION
**Pull request by paularah**
@xmulligan

### CODE REVIEW

File: src/pages/use-cases/service-mesh.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  description:
+    'Cilium Service Mesh redefines traditional service mesh frameworks by integrating the mesh layer directly into the kernel using eBPF, thus eliminating the need for sidecar proxies. It manages connectivity at both the networking and application protocol layers, handling protocols like IP, TCP, UDP, HTTP, Kafka, gRPC, and DNS with greater efficiency.  ',
+  videoSrc: 'https://www.youtube.com/embed/s-tgbD7wN3U',
+};
+
+const sectionContent2 = {
+  title: 'Superior Networking with eBPF',
+  description:
+    'At the heart of Cilium is eBPF, a revolutionary technology built into the Linux kernel. With eBPF, Cilium delivers lightning-fast, efficient, and scalable networking. This bypasses the performance drawbacks of traditional proxies, enabling direct and efficient communication between your services.',
+  imageSrc: ServiceMeshImage1,
```
Comment (hacktivist123) [inline src/pages/use-cases/service-mesh.jsx:31]: This image is missing, it breaks the page on my local, can you send a commit that adds this image?

Do you mean to name it servicemesh-1

Comment (paularah) [reply src/pages/use-cases/service-mesh.jsx:32]: Yes, I meant `servicemesh-1`

Comment (paularah) [reply src/pages/use-cases/service-mesh.jsx:32]: > This image is missing, it breaks the page on my local, can you send a commit that adds this image?
> 
> Do you mean to name it servicemesh-1

Yes, I meant `servicemesh-1`. I've pushed a fix for this.

**Comment by hacktivist123** [comment]
Hey @paularah Great job so far, I've reviewed your code and the content of the page. here are some observations that I discussed with @xmulligan 

- The runtime enforcement page, The "Security Observability and Runtime Enforcement with Cilium’s Tetragon" block should be on the right-hand side and the video should be on the left-hand side. This is based on the design I can see on Figma.

- Same with the "Kubernetes-Aware Real Time Enforcement" block on that same page. The text should be on the left and the image should be on the right.

- In the "Service Map" page, the text in the "Service Dependency Graph" section should be on the right and the image/video on the left

- In the "Metrics and tracing export" page, the image in the "Comprehensive & Rich Set Of Metrics" should be on the right and the text on the left. Same with the video and text in the next section of that same page

- In the "protocol visibility" page, the text in the "Gain Deep Insights Into Your Workload Communication At The Protocol Level With Cilium" section should be on the left and the image should be on the right.

- In the "BGP" page of the use cases, the text in the "Effortless Integration of Existing Network Infrastructure" section should be on the right and the image should be on the left.

These are all my observations for now, let me know if you need more clarity.

Also, I'll take a look at the fixes you made.

Great work!

**Comment by paularah** [comment]
> Hey @paularah Great job so far, I've reviewed your code and the content of the page. here are some observations that I discussed with @xmulligan
> 
> * The runtime enforcement page, The "Security Observability and Runtime Enforcement with Cilium’s Tetragon" block should be on the right-hand side and the video should be on the left-hand side. This is based on the design I can see on Figma.
> * Same with the "Kubernetes-Aware Real Time Enforcement" block on that same page. The text should be on the left and the image should be on the right.
> * In the "Service Map" page, the text in the "Service Dependency Graph" section should be on the right and the image/video on the left
> * In the "Metrics and tracing export" page, the image in the "Comprehensive & Rich Set Of Metrics" should be on the right and the text on the left. Same with the video and text in the next section of that same page
> * In the "protocol visibility" page, the text in the "Gain Deep Insights Into Your Workload Communication At The Protocol Level With Cilium" section should be on the left and the image should be on the right.
> * In the "BGP" page of the use cases, the text in the "Effortless Integration of Existing Network Infrastructure" section should be on the right and the image should be on the left.
> 
> These are all my observations for now, let me know if you need more clarity.
> 
> Also, I'll take a look at the fixes you made.
> 
> Great work!

Thank you! I made the designs on Figma so things aren't set in stone. I made some last-minute changes while building the pages that I felt were better aesthetically; that's why there are some differences between the Figma designs and the pages.

The rationale is to keep videos on the right as much as possible so it's easier to spot and consistent across pages, while images can float left or right. LMK if you think otherwise.

**Comment by hacktivist123** [comment]
> > Hey @paularah Great job so far, I've reviewed your code and the content of the page. here are some observations that I discussed with @xmulligan
> 
> > 
> 
> > * The runtime enforcement page, The "Security Observability and Runtime Enforcement with Cilium’s Tetragon" block should be on the right-hand side and the video should be on the left-hand side. This is based on the design I can see on Figma.
> 
> > * Same with the "Kubernetes-Aware Real Time Enforcement" block on that same page. The text should be on the left and the image should be on the right.
> 
> > * In the "Service Map" page, the text in the "Service Dependency Graph" section should be on the right and the image/video on the left
> 
> > * In the "Metrics and tracing export" page, the image in the "Comprehensive & Rich Set Of Metrics" should be on the right and the text on the left. Same with the video and text in the next section of that same page
> 
> > * In the "protocol visibility" page, the text in the "Gain Deep Insights Into Your Workload Communication At The Protocol Level With Cilium" section should be on the left and the image should be on the right.
> 
> > * In the "BGP" page of the use cases, the text in the "Effortless Integration of Existing Network Infrastructure" section should be on the right and the image should be on the left.
> 
> > 
> 
> > These are all my observations for now, let me know if you need more clarity.
> 
> > 
> 
> > Also, I'll take a look at the fixes you made.
> 
> > 
> 
> > Great work!
> 
> 
> 
> Thank you! I made the designs on Figma so things aren't set in stone. I made some last-minute changes while building the pages that I felt were better aesthetically; that's why there are some differences between the Figma designs and the pages.
> 
> 
> 
> The rationale is to keep videos on the right as much as possible so it's easier to spot and consistent across pages, while images can float left or right. LMK if you think otherwise. 
> 
> 
> 
> 

@paularah I see, that makes sense. I'm really just worried about inconsistency in the pages. If we are sticking to one style of positioning, let's just stick to that. I also noticed some pages where all the images and videos where all floated to the right. So maybe do a quick check and make sure there's consistency in the positioning of the images, text and videos on those pages.

**Comment by xmulligan** [comment]
I'm going to merge this so we can do a final check against the main branch

**Pull request merged**